msteams-platform/toolkit/deploy to the cloud for VS.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Deploy Teams app to the cloud using Visual Studio
+author: surbhigupta
+description:  In this module, learn how to deploy app to the cloud, Azure, or SharePoint and deploy Teams apps using Teams Toolkit in Visual Studio
+ms.author: v-amprasad
+ms.localizationpriority: medium
+ms.topic: overview
+ms.date: 11/29/2021
+---
+
+# Deploy Teams app to the cloud using Visual Studio
+
+Teams Toolkit helps you to deploy or upload your application's front-end and back-end code to the provisioned cloud resources in your Azure subscription. After deploying, you can preview the app in Teams client or the web browser before you can start using. The following apps can be deployed in Visual Studio:
+
+* The tab app, such as frontend applications are deployed to Azure storage, configured for static web hosting.
+* The notification bot app with Azure function triggers can be deployed to Azure functions.
+* The bot app or message extension can be deployed to Azure app services.
+
+## Prerequisite
+
+Here's a list of tools you need for building and deploying your apps.
+
+| &nbsp; | Install | For using... |
+| --- | --- | --- |
+| &nbsp; | **Required** | &nbsp; |
+| &nbsp; | Visual Studio | You can install the free community edition of Visual Studio, and install the workload. |
```

Comment:
Visual Studio 2022 v17.3

Reply:
You can install the enterprise edition of Visual Studio, and install the "ASP.NET "workload and Microsoft Teams Development Tools.

Reply:
These two suggestions have been implemented.